components/ProductCard.js

```diff
@@ -0,0 +1,36 @@
+import styled, { css } from "styled-components";
+
+export default function ProductCard({ name, quantity, category }) {
+  const categoryColors = {
+    Dairy: "pink",
+    Bakery: "wheat",
+    Fruits: "lightgreen",
+    Vegetables: "darkgreen",
+    Meat: "salmon",
+    Beverages: "lightgray",
+    Snacks: "skyblue",
+    Household: "lightyellow",
+    "Personal Care": "purple",
+    Other: "red",
+  };
+
+  const color = categoryColors[category];
+
+  return (
+    <>
+      <Card $categoryColor={color}>
+        <p>Name: {name}</p>
+        <p>Quantity: {quantity}</p>
+        <p>Category: {category}</p>
+      </Card>
+    </>
```

Comment:
You don't need the fragment here.

```suggestion
      <Card $categoryColor={color}>
        <p>Name: {name}</p>
        <p>Quantity: {quantity}</p>
        <p>Category: {category}</p>
      </Card>
```